dvol_test.go

```diff
@@ -1 +0,0 @@
-package main
```

Comment:
It's probably bad that I didn't know that was how gitHub showed that... we probably need to delete more flocker code...